pkg/types.go

```diff
@@ -20,10 +20,14 @@ package pkg
 
 import (
 	"fmt"
-	"github.com/jinzhu/gorm"
 	"time"
+
+	"github.com/jinzhu/gorm"
 )
 
+// Iso6801DateTime is the date format used in the API for denoting a zoned date time
+const Iso6801DateTime = "2006-01-02T15:04:05-07:00"
```

Comment:
Does this work with the way javascript stringifies Dates?
```
JSON.stringify(new Date())
"2019-12-03T10:13:21.999Z"
```

Reply:
without the millis